src/components/CustomStepper/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/styles';
+import Stepper from '@material-ui/core/Stepper'
+import Step from '@material-ui/core/Step'
+import StepLabel from '@material-ui/core/StepLabel'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+const useStyles = makeStyles(() => ({
+  resetButton:{
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  root: {
+    width: '90%'
+  },
+  buttonContainer: {
+    direction: 'rtl',
+    margin: '0% 4%',
+  },
+  button: {
+    textTransform: 'none',
+    color: '#7f7f7f',
+    '&.MuiButton-containedPrimary': {
+      color: '#fff',
+      backgroundColor: '#3EBBD3',
+      margin: '1%',
+    }
+  },
+  instructions: {
+    margin: '2% 5% 2%',
+  },
+  stepIconRoot: {
+    color: 'rgb(192 192 192)',
+    '&.MuiStepIcon-active': {
+      color: '#3EBBD3'
+    },
+    '&.MuiStepIcon-completed': {
+      color: '#82B982'
+    }
+  }
+}));
+
+const CustomStepper = ({ Content, Steps }) => {
+  const classes = useStyles();
+  const [state, setState] = useState({
+    activeStep: 0,
+    skipped: new Set()
+  })
+
+  const isStepOptional = (step) => {
+    return step === 1
+  }
+
+  const handleNext = () => {
+    const { activeStep } = state
+    let { skipped } = state
```

Comment:
declare skipped as a variable (let) instead of state destructuring

e.g: 
`let newSkipped = null `